tools/http3_test/tests/httpbin_tests.rs

```diff
@@ -25,6 +25,16 @@ mod httpbin_tests {
         url
     }
 
+    fn host() -> String {
+        let host = match std::env::var_os("HTTPBIN_HOST") {
+            Some(val) => val.into_string().unwrap(),
+
+            None => String::from("cloudflare-quic.com:443"),
```

Comment:
Do we always want users to set this when running the tests? An alternative would be to use `HTTPBIN_ENDPOINT` host by default, unless `HTTPBIN_HOST` is set. This way we also wouldn't need to hard-code cloudflare-quic.com in two places.

For example, `host()` could take the test URL as parameter, and return the value from that if `HTTPBIN_HOST` is not set.

Reply:
Makes sense, ta.

Reply:
done